src/cli/lint/autofix.ts

```diff
@@ -79,6 +82,21 @@ const shouldPush = async ({
     return false;
   }
 
+  if (currentBranch?.startsWith(RENOVATE_DEFAULT_PREFIX)) {
+    try {
+      await GitHub.getPullRequestNumber();
+    } catch (error) {
+      const warning =
+        'An autofix is available but it was not pushed because an open pull request for this skuba renovate branch could not be found. If a pull request has since been created, retry the lint step to push the fix.';
```

Comment:
```suggestion
        'An autofix is available, but it was not pushed because an open pull request for this Renovate branch could not be found. If a pull request has since been created, retry the lint step to push the fix.';
```